src/master/allocator/mesos/hierarchical.hpp

```diff
@@ -755,6 +755,8 @@ class HierarchicalAllocatorProcess : public MesosAllocatorProcess
 
   // Recovery data.
   Option<int> expectedAgentCount;
+  float agentRecoveryFactor;
```

Comment:
Is it possible to just use these values from `options`? Why keep a copy?

Reply:
My bad I reworked my patch and forgot to clean this.